docs_user/modules/proc_adopting-the-image-service.adoc

```diff
@@ -1,40 +1,36 @@
 [id="adopting-the-image-service_{context}"]
 
-//:context: adopting-image-service
-//kgilliga: This module might be converted to an assembly, or a procedure as a standalone chapter.
 //Check xref context.
+//Need to revisit this procedure. It will likely need to be split into multiple procedures: one for each backend. Do the verification steps apply to each backend?
 
-= Adopting the Image service
+= Adopting the {image_service}
 
-Adopting Glance means that an existing `OpenStackControlPlane` CR, where Glance
+Adopting {image_service_first_ref} means that an existing `OpenStackControlPlane` custom resource (CR), where {image_service}
 is supposed to be disabled, should be patched to start the service with the
 configuration parameters provided by the source environment.
 
 When the procedure is over, the expectation is to see the `GlanceAPI` service
-up and running: the `Keystone endpoints` should be updated and the same backend
-of the source Cloud will be available. If the conditions above are met, the
-adoption is considered concluded.
+up and running: the {identity_service} endpoints should be updated and the same backend of the source Cloud will be available. If the conditions above are met, the adoption is considered concluded.
```

Comment:
up and running: the {identity_service} endpoints are updated and the same backend of the source Cloud is available. If the conditions above are met, the adoption is considered concluded.

Reply:
Changed to present tense